exonum/src/node/state.rs

```diff
@@ -239,7 +251,8 @@ impl RequestData {
         #![cfg_attr(feature = "cargo-clippy", allow(match_same_arms))]
         let ms = match *self {
             RequestData::Propose(..) => PROPOSE_REQUEST_TIMEOUT,
-            RequestData::Transactions(..) => TRANSACTIONS_REQUEST_TIMEOUT,
+            RequestData::TransactionsForPropose(..) => TRANSACTIONS_REQUEST_TIMEOUT,
```

Comment:
Maybe `RequestData::TransactionsForPropose(..) | RequestData::TransactionsForBlock => TRANSACTIONS_REQUEST_TIMEOUT` to highlight the fact that those timeouts are the same?